src/main/java/com/softserveinc/ita/pageobjects/ProductPage.java

```diff
@@ -26,6 +27,14 @@ public BasketModal addToCart() {
         return new BasketModal();
     }
 
+    public int getProductPrice() {
+        return parseInt($x("//p[contains(@class, 'product-prices__big')]").getText().replaceAll("\\s|₴", ""));
```

Comment:
you able to create a method for such repeated actions

you could create NumberUtil with static method: parseIntPrice(String textPrice){
return parseInt(textPrice.replaceAll("\\s|₴", ""));
}

Reply:
take a look in the framework if somewhere else that such action with parsing a price is doing